libcalico-go/lib/backend/k8s/conversion/conversion.go

```diff
@@ -332,16 +346,26 @@ func (c converter) K8sNetworkPolicyToCalico(np *networkingv1.NetworkPolicy) (*mo
 		Types:    types,
 	}
 
-	// Build and return the KVPair.
-	return &model.KVPair{
+	// Build the KVPair.
+	kvPair := &model.KVPair{
```

Comment:
just a nit, but I think elsewhere when we have local variables for KVPairs we just name it `kvp`

Reply:
noted, will do!